theano/sandbox/cuda/cuda_ndarray.cu

```diff
@@ -3244,10 +3244,10 @@ CudaNdarray_gpu_init(PyObject* _unused, PyObject* args)
         if (cnmem > 1)
             mem = cnmem * 1024 * 1024;
         else{
-            // Clip to 98% to let memory for the driver.
-            // 98.5% didn't worked in some cases.
-            if (cnmem > .98){
-                cnmem = .98;
+            // Clip to 95% to let memory for the driver.
+            // 98% didn't worked in some cases.
+            if (cnmem > .99){
```

Comment:
Wait, that means if someone asks for 100%, they will get 95%, but if they ask for 99%, they will have 99%?
That seems unintuitive to me if what we want is to cap it at 95%.